api/spec/requests/v1/services_controller_spec.rb

```diff
@@ -138,7 +137,7 @@
                 identification_number: Service.last.identification_number,
                 name: Service.last.service_specification.name
               }
-            )
+            ).transform_values { |value| value.is_a?(Symbol) ? value.to_s : value }
```

Comment:
You could also use something like

```suggestion
            ).transform_values { |value| value.try(:to_s) || value }
```

Reply:
Can't do because then numbers would also get converted, which is not what this code should do.